k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -629,7 +638,13 @@ function deployCortxConfigMap()
     # Generate config files
     for i in "${!node_name_list[@]}"; do
         new_gen_file="$auto_gen_path/config.yaml"
-        cp "$cfgmap_path/templates/config-template.yaml" $new_gen_file
+        if [ $DEPLOYMENT_MODE -eq $STANDARD_DEPLOYMENT ];  
```

Comment:
Switch case is better for this instead of elif 

```
case $DEPLOYMENT_MODE in
    $STANDARD_DEPLOYMENT )
        cp "$cfgmap_path/templates/config-template.yaml" $new_gen_file
        ;;
    $DATA_ONLY_DEPLOYMENT )
        cp "$cfgmap_path/templates/config-data-template.yaml" $new_gen_file
        ;;
esac
```

Reply:
Fixed added switch case and tested